plugins/by-name/clangd-extensions/default.nix

```diff
@@ -1,271 +1,159 @@
-{
-  lib,
-  helpers,
-  config,
-  pkgs,
-  ...
-}:
-with lib;
+{ config, lib, ... }:
 let
-  cfg = config.plugins.clangd-extensions;
-
-  basePluginPath = [
-    "plugins"
-    "clangd-extensions"
-  ];
-
-  borderOpt = helpers.defaultNullOpts.mkBorder "none" "clangd-extensions" "";
+  inherit (lib.nixvim) defaultNullOpts;
+  inherit (lib) types mapAttrs;
 in
-{
-  # All of those warnings were introduced on 08/22/2023.
-  # TODO: Remove them in ~2 months (Oct. 2023).
-  imports =
-    [
-      (mkRemovedOptionModule (basePluginPath ++ [ "server" ]) ''
-        To configure the `clangd` language server options, please use
-        `plugins.lsp.servers.clangd.extraSettings`.
-      '')
-      (mkRemovedOptionModule (
-        basePluginPath
-        ++ [
-          "extensions"
-          "autoSetHints"
-        ]
-      ) "")
-    ]
```

Comment:
I know the original notice said "remove in ~2 months", but currently we're preferring to keep deprecations around until they cause issues.

Would you be able to move these removal notices to a separate module or something so that they're out of the way, but still exist? Like you've already done with the renamed-to-settings options.

If there's a legitimate reason to remove these notices, I'm open to that too. I'd just rather avoid removing them for the sake of it.

Reply:
Sure!